testcharms/charm-repo/bundle/kubernetes-simple/bundle.yaml

```diff
@@ -0,0 +1,10 @@
+bundle: kubernetes
+applications:
+    gitlab:
+        charm: gitlab
+        scale: 1
+        placement: foo=bar
+    mysql:
+        charm: mysql
```

Comment:
Shouldn't this be mariadb ;-)

Reply:
there needs to be a corresponding local charm. i've renamed the mysql one and updated usages in tests